src/tools/data/resolvers.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import { API_NAMESPACE } from './constants';
+import { setCronJobs } from './actions';
+
+export function* getCronJobs() {
+	const path = `${ API_NAMESPACE }/tools/get-cron-list/v1`;
+
+	try {
+		const response = yield apiFetch( {
+			path,
+			method: 'POST',
```

Comment:
Should we change the api method of this to a `GET`? given it only retrieves data and doesn't make any changes.

Reply:
Nice catch! I modified this in the commit 1089048